modules/lagoon/lagoon_search/lagoon_search.install

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * Update hooks for lagoon search.
+ */
+
+/**
+ * Ensures that the solr configuration is up-to-date.
+ *
+ * The latest version of solr changes the path and core, we need to ensure that
+ * all sites that are using Lagoon Search will have this updated. The install
+ * configuration reflects the correct configuration.
+ *
+ * @see https://github.com/govCMS/govcms8lagoon/pull/143
+ */
+function lagoon_search_update_8101() {
```

Comment:
The module can update the paths in update hook https://git.drupalcode.org/project/search_api_solr/-/blob/8.x-3.x/search_api_solr.install#L467

Reply:
When run against the canary that update hook either wasn't applied or didn't work as expected.

Reply:
@steveworley cool, thanks. For a few sites, they do have a search_api_solr server config file in config/default. So the old config got imported. This will be good then